app/Resources/views/full/bundle_list.html.twig

```diff
@@ -6,14 +6,56 @@
             <div class="container">
                 <div class="row">
                     <div class="col-xs-12">
-                        {{ ez_render_field(content, 'description') }}
+                        <h1>{{ ez_render_field(content, 'name') }}</h1>
+                        <h2>{{ ez_render_field(content, 'description') }}</h2>
                     </div>
                 </div>
             </div>
         </div>
+        <div class="bundles-list-search">
+            <div class="container">
+                <div class="col-xs-12">
+                    {{ form_start(searchForm) }}
+                        <div class="form-group">
+                            {{ form_widget(searchForm.search, { 'id': 'bundles-list-search-query'} )}}
+                            <div class="input-group-addon">
+                                <button type="submit" class="btn bundles-list-search-button">
+                                    <i class="fa fa-search" aria-hidden="true"></i>
+                                </button>
+                            </div>
+                        </div>
+                    {{ form_end(searchForm) }}
+                </div>
+            </div>
+        </div>
         <div class="bundles-list-content">
             <div class="container">
-                <div class="col-xs-12 bundles-list-container">
+                <div class="hidden-xs hidden-sm col-md-2 bundle-list-settings">
```

Comment:
I think this part can be removed. This feature will be implemented on a different PR